### PR TITLE
electron-devtools-installer: Allow arrays of extensionreferences

### PR DESCRIPTION
**Pull request by aeons**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/MarshallOfSound/electron-devtools-installer/blob/0b07982a063079aa12ffc34c12c202e7061c0a62/src/index.js#L22
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/electron-devtools-installer/electron-devtools-installer-tests.ts [hunk 1]
```diff
@@ -1,11 +1,4 @@
-import installExtension, {
-    EMBER_INSPECTOR, REACT_DEVELOPER_TOOLS,
-    BACKBONE_DEBUGGER, JQUERY_DEBUGGER,
-    ANGULARJS_BATARANG, VUEJS_DEVTOOLS,
-    REDUX_DEVTOOLS, REACT_PERF,
-    CYCLEJS_DEVTOOL, APOLLO_DEVELOPER_TOOLS,
-    MOBX_DEVTOOLS
-} from 'electron-devtools-installer';
+import installExtension, { ANGULARJS_BATARANG, APOLLO_DEVELOPER_TOOLS, BACKBONE_DEBUGGER, CYCLEJS_DEVTOOL, EMBER_INSPECTOR, JQUERY_DEBUGGER, MOBX_DEVTOOLS, REACT_DEVELOPER_TOOLS, REACT_PERF, REDUX_DEVTOOLS, VUEJS_DEVTOOLS } from 'electron-devtools-installer';
```
Comment (MFatihMAR) [inline types/electron-devtools-installer/electron-devtools-installer-tests.ts]: You shouldn't change formatting.

File: types/electron-devtools-installer/index.d.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ declare module "electron-devtools-installer" {
     }
 
     // Default installation function
-    export default function (extensionReference: ExtensionReference | string, forceDownload?: boolean): Promise<string>;
+    export default function (extensionReference: ExtensionReference | string | Array<ExtensionReference | string>, forceDownload?: boolean): Promise<string>;
```
Comment (MFatihMAR) [inline types/electron-devtools-installer/index.d.ts]: I think it's not necessary to support array as they are already a few.

**Review by 0xFA11 — COMMENTED** [review]
I'm not quite sure about changes, so I'm going to leave it for somebody else to review. Hopefully they'll decide more precisely!

File: types/electron-devtools-installer/electron-devtools-installer-tests.ts (hunk 1 above)
Comment (aeons) [reply types/electron-devtools-installer/electron-devtools-installer-tests.ts]: That was some trigger happy editor plugin, reverted it now.

File: types/electron-devtools-installer/electron-devtools-installer-tests.ts [hunk 3]
```diff
@@ -19,4 +12,7 @@ installExtension(REACT_PERF);
 installExtension(CYCLEJS_DEVTOOL);
 installExtension(APOLLO_DEVELOPER_TOOLS);
 installExtension(MOBX_DEVTOOLS);
-installExtension('abcdefghijkl');
\ No newline at end of file
+installExtension('abcdefghijkl');
+installExtension([EMBER_INSPECTOR, JQUERY_DEBUGGER]);
```
Comment (MFatihMAR) [inline types/electron-devtools-installer/electron-devtools-installer-tests.ts]: As a side-request, while you're already covering Ember+jQuery and React+MobX extension installations with an array, can you simply remove installExtension(EMBER...), jQuery, React, MobX tools from above please (and see what happens)?
This way we can have much more meaningful test script here I believe.

**Pull request merged**